Worksheet B - Pseudocode.txt

```diff
@@ -0,0 +1,17 @@
+Import random library
+N = Random INT from 1-10
+
+Word = Random String of N Characters
+
+Guess = input ("What is the secret word, if it has " + N + "letters?")
+
+For N until Guess[N] or until after fourth guess:
+	If Guess == Word:
+		Print "You have done it correctly!"
+	If Guess == Word - 1 character:
```

Comment:
The worksheet asked for pseudocode to compute the number of correct letters -- be sure to read specifications carefully in future!

Reply:
Not clear what you mean by `Guess == Word - 1 character` here -- someone not familiar with the terminal hacking game would have trouble interpreting this correctly. Try to be more precise and leave less room for ambiguity.